### PR TITLE
filter doc + support for scala highlighting

### PR DESCRIPTION
**Pull request by anshuman-91**
Filter doc

### CODE REVIEW

File: docs/05-low-code-spark/05-gems/02-transform/01-reformat.md [hunk 1]
```diff
@@ -3,8 +3,71 @@ sidebar_position: 1
...
 
+### Parameters
+| Parameter        | Meaning                                       | Required                                     |
+|:-----------------|:----------------------------------------------|:---------------------------------------------|
+| Dataframe        | Input dataframe on which changes are required | True                                         |
+| Target column    | Output column name                            | False                                        |
+| Expression       | Expression to compute target column           | False (Required if target column is present) |
+
+:::info 
+To quickly select all the columns, click on the ADD ALL on the left side bar.
```
Comment (slyons) [inline docs/05-low-code-spark/05-gems/02-transform/01-reformat.md:17]: Screenshot here would be handy

File: docs/05-low-code-spark/05-gems/02-transform/01-reformat.md [hunk 2]
```diff
@@ -3,8 +3,71 @@ sidebar_position: 1
...
+:::info 
+To quickly select all the columns, click on the ADD ALL on the left side bar. 
+:::
+:::info
+If no columns are selected, then all columns are passed through to the output
 :::
+
+### Example
+
+![](./img/reformat_eg_1.png)
```
Comment (slyons) [inline docs/05-low-code-spark/05-gems/02-transform/01-reformat.md:25]: Always include Alt text (inside the square brackets) for accessibility reasons

File: docusaurus.config.js [hunk 3]
```diff
@@ -92,6 +92,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages: ['java','scala'],
```
Comment (slyons) [inline docusaurus.config.js:95]: Do we need Python here?

**Review by slyons — COMMENTED** [review]
Looks good so far. We need more examples for each Gem and the resulting outputs for each example

File: docusaurus.config.js (hunk 3 above)
Comment (anshuman-91) [reply docusaurus.config.js:95]: no, it has inbuilt support for python, unlike scala. So, just had to add scala and java(for scala)

File: docs/05-low-code-spark/05-gems/02-transform/01-reformat.md (hunk 1 above)
Comment (anshuman-91) [reply docs/05-low-code-spark/05-gems/02-transform/01-reformat.md:17]: makes sense

File: docs/05-low-code-spark/05-gems/02-transform/01-reformat.md (hunk 2 above)
Comment (anshuman-91) [reply docs/05-low-code-spark/05-gems/02-transform/01-reformat.md:25]: fair

**Pull request merged**